packages/p2p/src/components/buy-sell/rate-change-modal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Modal, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize, Localize } from 'Components/i18next';
+import 'Components/buy-sell/rate-change-modal.scss';
```

Comment:
can just use `./rate-change-modal.scss`?

Reply:
will change the path